scipy/stats/_morestats.py

```diff
@@ -3910,20 +3910,23 @@ def directionalmean(samples, *, axis=0, normalize=True):
     """
     Computes the directional mean of a sample of vectors.
 
-    Serves as equivalent of the sample mean for directional data whose
-    magnitude is irrelevant, e. g. unit vectors. The vectorial data
-    whose directional mean are computed must be located in the last
-    axis.
+    The directional mean is a measure of "preferred direction" of vector data.
+    It is analogous to the sample mean, but it is for use when the magnitude
+    the data is irrelevant (e.g. unit vectors).
 
     Parameters
     ----------
     samples : array_like
-        Input array. Must be at least two-dimensional.
-    axis : int, optional
-        Axis along which directional means are computed. Default is 0.
-    normalize: boolean, optional
-        If True, normalize the input data in the last axis
-        to unit vectors of norm 1. Default is True.
+        Input array. Must be at least two-dimensional, and the last axis of the
+        input must correspond with the dimensionality of the vector space.
+        When the input is exactly two dimensional, this means that each row
+        of the data is a vector observation.
+    axis : int, default: 0
```

Comment:
It is OK to put the default in the text, but I think it is easier to find and less verbose when the default is listed next to the data type.